tests/unit/test_pytorch.py

```diff
@@ -582,33 +580,33 @@ def test_model_py2_warning(warning, sagemaker_session, pytorch_version):
 
 def test_pt_enable_sm_metrics(sagemaker_session, pytorch_full_version):
     pytorch = _pytorch_estimator(
-        sagemaker_session, framework_version=pytorch_full_version, enable_sagemaker_metrics=True
+        sagemaker_session, framework_version=pytorch_full_version, py_version="py3", enable_sagemaker_metrics=True
     )
     assert pytorch.enable_sagemaker_metrics
 
 
 def test_pt_disable_sm_metrics(sagemaker_session, pytorch_full_version):
     pytorch = _pytorch_estimator(
-        sagemaker_session, framework_version=pytorch_full_version, enable_sagemaker_metrics=False
+        sagemaker_session, framework_version=pytorch_full_version, py_version="py3", enable_sagemaker_metrics=False
     )
     assert not pytorch.enable_sagemaker_metrics
 
 
 def test_pt_disable_sm_metrics_if_pt_ver_is_less_than_1_15(sagemaker_session):
     for fw_version in ["1.1", "1.2"]:
-        pytorch = _pytorch_estimator(sagemaker_session, framework_version=fw_version)
+        pytorch = _pytorch_estimator(sagemaker_session, framework_version=fw_version, py_version="py3")
         assert pytorch.enable_sagemaker_metrics is None
 
 
 def test_pt_enable_sm_metrics_if_fw_ver_is_at_least_1_15(sagemaker_session):
     for fw_version in ["1.3", "1.4", "2.0", "2.1"]:
-        pytorch = _pytorch_estimator(sagemaker_session, framework_version=fw_version)
+        pytorch = _pytorch_estimator(sagemaker_session, framework_version=fw_version, py_version="py3")
         assert pytorch.enable_sagemaker_metrics
 
 
 def test_custom_image_estimator_deploy(sagemaker_session, pytorch_full_version):
     custom_image = "mycustomimage:latest"
-    pytorch = _pytorch_estimator(sagemaker_session, framework_version=pytorch_full_version)
+    pytorch = _pytorch_estimator(sagemaker_session, framework_version=pytorch_full_version, py_version="py3")
```

Comment:
any reason not to use `pytorch_py_version` fixture?

Reply:
I'd chosen not to use it because `pytorch_full_version` provides only the latest version, and `pytorch_py_version` doesn't correspond with it. however, now I've changed it so that the test uses `pytorch_version` and `pytorch_py_version`